openlibrary/plugins/openlibrary/lists.py

```diff
@@ -353,6 +353,23 @@ def POST(self, user_key: str | None, list_key: str | None = None):  # type: igno
             return safe_seeother(list_record.key)
 
 
+class lists_add_account(delegate.page):
+    path = r"/account/lists/add.*"
```

Comment:
This should auto match the query parameters without the `.*` :+1: 
```suggestion
    path = r"/account/lists/add"
```